srcpkgs/gimp/template

```diff
@@ -60,7 +60,7 @@ gimp-python_package() {
 	pycompile_dirs="usr/lib/gimp/2.0"
 	short_desc+=" - Python2 bindings"
 	pkg_install() {
-		vmove usr/lib/gimp/2.0/python
-		vmove "usr/lib/gimp/2.0/plug-ins/*.py"
+		vmove "usr/lib/gimp/2.0/python"
```

Comment:
no globs no need for quoting

Reply:
can remove the quotes

https://github.com/void-linux/void-packages/blob/master/Manual.md#namingconvention
(the # doesnt seem to work but anyway) not sure what you mean. guessing you want the branch name to be gimp-2.10.8 ?

Reply:
The commit message should be `gimp: update to 2.10.8` 

https://github.com/void-linux/void-packages/blob/master/Manual.md#contributing